functional/security/Crypto/build.xml

```diff
@@ -52,31 +52,31 @@
 		<mkdir dir="${DEST}" />
 	</target>
 	<if>
-		<or>
-			<matches pattern="^[89]{1}$" string="${JDK_VERSION}"/>
-			<matches pattern="^[1][023456]" string="${JDK_VERSION}"/>
-		</or>
+		<matches pattern="^([89]|1[02-6])$" string="${JDK_VERSION}"/>
 		<then>
 			<property name="jtregTar" value="jtreg_5_1_b01"/>
 		</then>
+		<elseif>
+			<matches pattern="^11$" string="${JDK_VERSION}"/>
+			<then>
+				<property name="jtregTar" value="jtreg_6_1"/>
+			</then>
+		</elseif>
+		<elseif>
+			<matches pattern="^1[89]$" string="${JDK_VERSION}"/>
+			<then>
+				<property name="jtregTar" value="jtreg_6_1_1"/>
+			</then>
+		</elseif>
```

Comment:
Since JDK18 and 19 are out of support, could you remove this if block?

Reply:
Fixed in a7d41bb7ceb6be3f62d1af2831f937330a59fdd6.